src/bundle/Resources/views/themes/admin/ui/field_type/preview/content_fields.html.twig

```diff
@@ -175,15 +175,21 @@
         {% set attr = attr|merge({'class': (attr.class|default('') ~ ' ibexa-field-preview ibexa-field-preview--ezselection-multiple')|trim}) %}
         <ul {{ block( 'field_attributes' ) }}>
         {% for selectedIndex in field.value.selection %}
-            <li class="ibexa-selection__item">{{ options[selectedIndex] }}</li>
+            <li class="ibexa-selection__item ibexa-tag">
+                <div class="ibexa-tag__content">
+                    {{ options[selectedIndex] }}
+                </div>
+            </li>
         {% endfor %}
         </ul>
     {% elseif not fieldSettings.isMultiple %}
-        {% set attr = attr|merge({'class': (attr.class|default('') ~ ' ibexa-field-preview ibexa-field-preview--ezselection-single')|trim}) %}
+        {% set attr = attr|merge({'class': (attr.class|default('') ~ ' ibexa-field-preview ibexa-field-preview--ezselection-single ibexa-tag')|trim}) %}
 
         {% set field_value = options[field.value.selection|first]|escape %}
 
-        {{ block( 'simple_block_field' ) }}
+        <div class="ibexa-tag__content">
```

Comment:
I think there's missing `ibexa-tag` class somewhere here?

Reply:
this class is set in attr variable in line 186
https://github.com/ibexa/admin-ui/pull/554/files/61e5ec00e7438b1eaebcc64668ca07ed9ebbbbde#diff-03a88b29d0af2169f521b6db3be86937e4791cfa8d844d022acd2a3646927d96R186